spec/models/product_spec.rb

```diff
@@ -1,5 +1,67 @@
 require 'rails_helper'
 
 RSpec.describe Product, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:category) {
+    Category.create!(
+      name: "asdasdas"
+    )
+  }
+
+  let(:user) {
+    User.create(
+      fname: "bago",
+      lname: "luma",
+      mobile: 123_456_789,
+      email: "sample@yahoo.com",
+      birthdate: "2018-01-07",
+      role: "admin",
+      password: "foobar",
+      avatar:
+      Rack::Test::UploadedFile.new(
+        Rails.root.join('spec', 'support', '1.png')
+      )
+    )
+  }
+
+  let(:valid_attributes) {
+    {
+      user_id: user.id,
+      name: "Cap",
+      description: "Black",
+      price: 90.75,
+      category_id: category.id,
+      image: Rack::Test::UploadedFile.new(
+        Rails.root.join('spec', 'support', '1.png')
+      )
+    }
+  }
+
+  describe Product do
+    it "should belongs to user" do
+      relationship = Product.reflect_on_association(:user)
+      expect(relationship.macro).to eq(:belongs_to)
+    end
+  end
+
+  describe Product do
+    it "should belongs to category" do
```

Comment:
Bawal ba siya encapsulate, sa loob ng  https://github.com/jaimejazarenoiii/CMS-RSpec/pull/1/files#diff-817905e2c2c6d60a151a80856af66b9dR39 since same naman sila `describe Product`